sql/database.sql

```diff
@@ -10175,9 +10176,8 @@ CREATE TABLE IF NOT EXISTS `medex_prefs` (
   `LABELS_local` varchar(3) DEFAULT NULL,
   `LABELS_choice` varchar(50) DEFAULT NULL,
   `combine_time` tinyint(4) DEFAULT NULL,
-  `notify` varchar(100) DEFAULT NULL,
   `postcard_top` varchar(255) DEFAULT NULL,
-  `MedEx_lastupdated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
This seems odd since you don't make this change in above sql/5_0_0-to-5_0_1_upgrade.sql script